### ISSUE TITLE
HTML does not show \pnum

### ISSUE THREAD

**Issue by Axel-Naumann** [mpark/wg21#20]
`\pnum{}`s are swallowed by HTML output. I use HTML output as CWG really wants to have strikethrough/underline, which doesn't work in PDF, see #19 . But not having paragraph numbers will make CWG at least as unhappy... Whatever is easier to fix (also taking into account #18 - maybe adding underline / strikethrough to PDF is simpler after all?)

**Comment by Axel-Naumann** [comment]
This?
```
diff --git a/template/diff.py b/template/diff.py
index 0369ea6..2ec922c 100755
--- a/template/diff.py
+++ b/template/diff.py
@@ -27,8 +27,12 @@ More new paragraphs
 """

 import panflute as pf
+import re

 def action(elem, doc):
+    if isinstance(elem, pf.RawInline) and elem.format == 'tex' and '\pnum{' in elem.text:
+        repl = re.sub(r"\\pnum{([^}]+)}", "<small>(\\1)</small>", elem.text)
+        return pf.RawInline(repl, 'html')
     if not isinstance(elem, pf.Div) and not isinstance(elem, pf.Span):
         return None
```

**Comment by mpark** [comment]
This is fixed by a `pnum` span element introduced in 315381b.
Specify the paragraph number like this:`[2]{.pnum}`.

Given the following markdown:
```markdown
[2]{.pnum} _Requires:_ For each valid pack `m` _e_(`m`) shall be a valid
expression. All such expressions shall be of the same type and value category;
otherwise, the program is ill-formed.

[3]{.pnum} _Returns:_ _e_(`m`), where `m` is the pack for which `m`_~i~_ is
`vars`_~i~_`.index()` for all $0 \leq i < n$. The return type is
`decltype(`_e_(`m`)`)` [for the first form]{.add}.
```

The generated HTML is:

![Screen Shot 2019-05-29 at 4 34 42 AM](https://user-images.githubusercontent.com/2275240/58553993-1a3c1480-81cb-11e9-8582-605290d63c88.png)
